dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -250,9 +250,34 @@ public String getAlternateInForwardStrand() {
         }
     }
 
+    /**
+     * Return the field "alleles" in forward strand.
+     *
+     * The 3 orientations (snp_orientation, contig_orientation, subsnp_orientation) are relevant to put the field
+     * "alleles" (which comes from the column obsvariation.pattern) in the forward strand.
+     *
+     * As example, look at the next rs, using the hgvs strings and orientations to know if the
+     * ref_allele_c, ref_allele_t and alternate are forward or reverse, thus
+     * knowing if "alleles" is in forward or reverse.
+     *
+     * - rs13677177 : "alleles" are in forward when orientations are 1 1 1, reverse when orientations 1 1 -1
+     * - rs739617577 : "alleles" are in forward with orientations -1 -1 1
+     * - rs10721689 :  "alleles" are reverse when orientations are 1 -1 1, forward when 1 -1 -1
+     */
     public String getAllelesInForwardStrand() {
+        int forward = 1;
```

Comment:
Given the large number of variants that will be processed, it could be more efficient to change a boolean than multiplying several times.